src/routes/financialData.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+
+import { verifyOtherDebs } from '../controllers/financialData.controller';
+import { financialDataDebtsValidator } from '../validators/financialData.validator';
+
+const router = express.Router();
+
+router.post('/set-other-debts', financialDataDebtsValidator, verifyOtherDebs);
```

Comment:
i think you have to manage messages, see how is used in other controllers in user